cool/vars/production.tfvars

```diff
@@ -1,10 +1,7 @@
 #=================================================
 #  PROVIDER
 #=================================================
-github_actions_role = "arn:aws:iam::779871898657:role/ProvisionAccount"
-shared_services_role = "arn:aws:iam::121087745329:role/ProvisionPrivateDNSRecords"
 workspace_type = "production"
-account_id = "779871898657"
```

Comment:
Can the account ID also be removed from the Docker `image_url` below and in `staging.tfvars`?

Reply:
@dav3r Fixed in https://github.com/cisagov/domain-manager-cicd/pull/11/commits/b6bed6d26d8035e2635f150c25bf451dd27d2276